mantidproject/mantid
# 16291 eqsans gui bug

Bug Fix. Minimal changes to correct a bad import and field in the GUI rendered as property not number.

No release Notes needed or documentation.

**To test:**



Fixes #16291.

1- Test opening the file in the EQSANS GUI (Debug Mode On): /SNS/EQSANS/IPTS-15454/shared/67315_Iq.xml.

The Detector off set should show 0.
Reduce shouldn't throw any errors.

2- Re-reduce that file, and check that you can reload the new xml file.

3- Check that it also works with the HFIR UI.



---
#### Reviewer

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):
##### Code Review
##### Functional Tests
- Does everything look good? Comment with the ship it emoji but don't merge. A member of `@mantidproject/gatekeepers` will take care of it.
